tests/test_pagegen.py

```diff
@@ -158,3 +173,33 @@ def test_show_both_commits_in_description():
     f = open('./tests/.out/test_show_both_commits_in_description.md', 'r')
     file_content = f.read()
     assert 'Results of [`614c013`](https://github.com/chocoteam/choco-solver/commit/614c0134750071ffe08dc376e9cc8caf210974bf) are compared with [`13a4c1d`](https://github.com/chocoteam/choco-solver/commit/13a4c1dca0dd58d62acc741866fb945f3fe81592).' in file_content
+
+def test_metadata_are_used_to_generate_front_matter():
+    tests_output_file_name: str = 'test_metadata_are_used_to_generate_front_matter.out'
+    commit1: str = '13a4c1dca0dd58d62acc741866fb945f3fe81592'
+    commit2: str = '614c0134750071ffe08dc376e9cc8caf210974bf'
+    output_file_folder: str = './tests/.out/'
+    input_data = PageGenInputData(
+        Metadata(
+            test_folder_path='/home/evaluation/evaluation/pub/bench/',
+            page_title='Title',
+            page_description='Description',
+            repository_url='http://wesite.com/repository/',
+            similar_percent_limit=50
+        ),
+        [
+            TestResult(
+                '/home/evaluation/evaluation/pub/bench/XCSP18/CrosswordDesign/CrosswordDesign-03-4-rom_c18',
+                Diff('Exit value 1', 50, 40, -10, -20),
+                []
+            ),
+        ],
+    )
+
+    generate_page(tests_output_file_name, commit1, commit2, output_file_folder, input_data)
+    f = open('./tests/.out/test_metadata_are_used_to_generate_front_matter.md', 'r')
```

Comment:
Same (assertions on file existence and on file size > 0)

Reply:
I use the file content here, doesn't apply